examples/demo-nft-module/tests/nft_test.rs

```diff
@@ -1,159 +1,452 @@
-use demo_nft_module::{CallMessage, NonFungibleToken, NonFungibleTokenConfig, OwnerResponse};
```

Comment:
We should add integration tests in `demo-rollup/tests`. The tests there are higher level and use real rpc and mock DB. It can be done as a separate PR.

Reply:
https://github.com/Sovereign-Labs/sovereign-sdk/issues/871